serverless/aws/lambda/src/main/scala/sttp/tapir/serverless/aws/lambda/runtime/AwsLambdaRuntime.scala

```diff
@@ -0,0 +1,113 @@
+package sttp.tapir.serverless.aws.lambda.runtime
+
+import cats.effect.{Blocker, ConcurrentEffect, ContextShift, Resource}
+import cats.syntax.either._
+import com.typesafe.scalalogging.StrictLogging
+import io.circe.Printer
+import io.circe.generic.auto._
+import io.circe.parser.decode
+import io.circe.syntax._
+import org.http4s.client.blaze.BlazeClientBuilder
+import sttp.client3._
+import sttp.client3.http4s.Http4sBackend
+import sttp.monad.MonadError
+import sttp.monad.syntax._
+import sttp.tapir.integ.cats.CatsMonadError
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.serverless.aws.lambda._
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.DurationInt
+
+// loosely based on https://github.com/carpe/scalambda/blob/master/native/src/main/scala/io/carpe/scalambda/native/ScalambdaIO.scala
+abstract class AwsLambdaRuntime[F[_]] extends StrictLogging {
+  implicit def executionContext: ExecutionContext
+  implicit def contextShift: ContextShift[F]
+  implicit def concurrentEffect: ConcurrentEffect[F]
+
+  implicit def serverOptions: AwsServerOptions[F] = AwsServerOptions.customInterceptors()
+  def endpoints: Iterable[ServerEndpoint[_, _, _, Any, F]]
+
+  protected val backend: Resource[F, SttpBackend[F, Any]] = Http4sBackend.usingBlazeClientBuilder(
+    BlazeClientBuilder[F](executionContext).withConnectTimeout(0.seconds),
+    Blocker.liftExecutionContext(implicitly)
+  )
+
+  def main(args: Array[String]): Unit = {
+    val route: Route[F] = AwsServerInterpreter.toRoute(endpoints.toList)
+    implicit val monad: MonadError[F] = new CatsMonadError[F]
+
+    val runtimeApiInvocationUri = uri"http://${sys.env("AWS_LAMBDA_RUNTIME_API")}/2018-06-01/runtime/invocation"
+    val nextEventRequest = basicRequest.get(uri"$runtimeApiInvocationUri/next").response(asStringAlways).readTimeout(0.seconds)
```

Comment:
why do we have 0 sec timeout here?

Reply:
It is handled this way in Scalambda, they say that:
```
Make request (without a timeout as prescribed by the AWS Custom Lambda Runtime documentation). 
This is due to the possibility of the runtime being frozen between lambda function invocations. 
Continuously polling also seems to cause issues with AWS's internal lambda state management service,
so this definitely seems to be the best way of fetching the event.
```
I've searched in lambda custom runtimes documentation but found no explicit mention about timeouts there.

Reply:
ah, so 0 timeout is mean to disable it. Ok, let's add a comment maybe where this comes from?